client/locations-list-map.js

```diff
@@ -1442,6 +1448,25 @@ const applyFilterParams = ((params, filterSet) => {
   });
 });
 
+const initializeEmbedLocationCollapse = () => {
```

Comment:
thoughts on trying to remove jQuery dependency at some point? I think we were trying to get there but it might just not be worth it at this point given that we use bootstrap's popover / modal now

Reply:
Hahah, yeah, I try not to use jQuery in my "real" life these days but I'm not sure it's worth us trying to escape it for this project, esp. given your point about Bootstrap JS